src/components/shared/AboutDialog/AboutDialog.tsx

```diff
@@ -21,10 +27,9 @@ const AboutDialog: FC<AboutDialogProps> = ({open, onClose}) => {
                     Hours was created by Nathan Luu, Nick Young, and Neil Ramaswamy at Brown University to facilitate
                     office hours for courses in the CS Department and beyond.
                 </Typography>
-                {/*<Button startIcon={<GitHubIcon/>} color="inherit" variant="outlined"*/}
-                {/*        href="https://github.com/nthnluu/hours-frontend/wiki/Contributing">*/}
-                {/*    Contribute*/}
-                {/*</Button>*/}
+                <Button startIcon={<GitHubIcon/>} color="inherit" variant="outlined" onClick={() => handleExternalClick("https://github.com/nthnluu/hours-frontend/wiki/Contributing")}>
```

Comment:
as before, use the `href` prop alongside `target="_blank" rel="noopener noreferrer"` to open in a new tab